app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,9 @@
         <activity
             android:name="org.fossasia.openevent.general.auth.LoginActivity">
         </activity>
+        <activity
+            android:name="org.fossasia.openevent.general.auth.SignUpActivity">
+        </activity>
```

Comment:
Login and Signup should should be fragments in a single activity

Reply:
Okay:+1: